### PR TITLE
Fix homepage link

### PR DESCRIPTION
**Pull request by zackkrida**
Updates the broken homepage link in `package.json` to fix our public homepage link on npm. Will be fixed the next time we release.

~Note: Is technically blocked by the next deployment of https://opensource.creativecommons.org (I don't know what the deployment process there is.)~ Okay, that's fast. Nice! 😆 

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by kgodey** [comment]
The Open Source @ CC website autodeploys on merge to `master` using GitHub Actions to build the site, and then that gets auto-deployed by GitHub Pages.

**Comment by dhruvkb** [comment]
I actually meant you could add a cc- to the path and skip adding the redirect at all.

File: package.json [hunk 1]
```diff
@@ -52,7 +52,7 @@
     "webpack-cli": "^3.3.10"
   },
   "bugs": "https://github.com/creativecommons/vocabulary/issues/",
-  "homepage": "https://creativecommons.github.io/vocabulary/",
+  "homepage": "https://opensource.creativecommons.org/vocabulary",
```
Comment (dhruvkb) [inline package.json:55]: If you change it to this, you wouldn't even need the redirect?
```suggestion
  "homepage": "https://opensource.creativecommons.org/cc-vocabulary/",
```

**Review by dhruvkb — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (zackkrida) [reply package.json:55]: https://opensource.creativecommons.org/vocabulary is already live and working too. the redirects for `/vocabulary` and `/cc-vocabulary` are the same.

**Review by dhruvkb — APPROVED** [review]
I'll approve it (if you want to merge this PR) as it makes no significant difference either way, though _I'd still prefer if we used `/cc-vocabulary` as the path_, if only for maintaining consistency with the other places we've used this link.

**Comment by kgodey** [comment]
I think deprecating `cc-vocabulary` makes sense, this repo used to be named that and that's why we had the original link. I'd rather be consistent with the current state of things.

**Pull request merged**